app/ServiceProvider/ActionProvider.php

```diff
@@ -98,8 +100,7 @@ public function register(Container $container)
         $container['actionManager']->register(new TaskAssignColorSwimlane($container));
         $container['actionManager']->register(new TaskAssignPrioritySwimlane($container));
         $container['actionManager']->register(new TaskAssignColorOnDueDate($container));
-        $container['actionManager']->register(new SubtaskTimerMoveTaskColumn($container));
-        $container['actionManager']->register(new StopSubtaskTimerMoveTaskColumn($container));
```

Comment:
Why did you remove these actions?

Reply:
Hi, 

This was an error, I fixed it, sorry